schema.sql

```diff
@@ -102,3 +102,14 @@ CREATE TABLE visits (
     ON DELETE CASCADE
 );
 
+/*Milestone 4*/
+
```

Comment:
- [ ] According to the project's requirements, kindly make sure that `INSERT INTO` queries are saved in the `data.sql` file, as right now you didn't add them to any of the existed **files**.

Reply:
We just done it